Assets/Scripts/Model/RollModifier/AbstractModifier.cs

```diff
@@ -5,25 +5,44 @@ namespace Fumbbl.Model.RollModifier
     public abstract class AbstractModifier : FfbEnumerationFactory
     {
         public int Modifier { get; set; }
+        public bool ForcedModifiedString { get; set; } = false;
```

Comment:
This is not an elegant solution. The functional equivalent would be to have a "SignIncludedInName" property together with the two other configuration properties.

It's probably a good idea to add a straight-up format datatype to group these settings:
struct Format {
  bool ShowSign;
  bool ShowModifier;
  bool ShowName;
  bool ReverseModifier;
}

Makes the processing clearer and more generic in nature.